pinot-integration-tests/src/test/java/org/apache/pinot/integration/tests/MergeRollupMinionClusterIntegrationTest.java

```diff
@@ -306,6 +306,8 @@ public void testSingleLevelConcat()
     // Check total tasks
     assertEquals(numTasks, 5);
 
+    Assert.assertTrue(_controllerStarter.getControllerMetrics().containsGauge("myTable1_OFFLINE.100days"));
```

Comment:
nit: you don't need the `Assert.` prefix if `assertTrue` shows up as static import, similarly to other assertion methods in the test.

Reply:
I changed the test to make it consistent.  I removed `Assert.`.